### PR TITLE
GraphQL

### PR DESCRIPTION
**Pull request by zachstence**
Changes:
- Change some annoying ESLint rules
- Add GQL URL to config
- Add embed service and `embed orgId` command to test
- Reimplement `getSprocketConfig key` command using GQL
- Reimplement member lookup/registration to use GQL
- Remove `CoreService` RMQ client injection

### CODE REVIEW

**Review by 0c370t — CHANGES_REQUESTED** [review]

File: src/marshal/types.ts [hunk 1]
```diff
@@ -9,7 +8,9 @@ export enum MarshalMetadataKey {
 
 export interface MarshalCommandContext {
     args?: Record<string, string>;
-    author: Omit<UserModel, "authenticationAccounts"> | false;
+    author: {
+        id: string;
```
[the remark this answers is not in the document]
Comment (zachstence) [reply src/marshal/types.ts:12]: Yeah, just did enough for now. We should address in the future

File: src/marshal/marshal.ts [hunk 2]
```diff
@@ -17,10 +16,10 @@ import {
 export abstract class Marshal {
     private readonly __logger__ = new Logger(Marshal.name);
 
-
     constructor(
         _cms: CommandManagerService,
-        protected readonly coreService: CoreService,
+        protected readonly embedService: EmbedService,
```
[the remark this answers is not in the document]
Comment (zachstence) [reply src/marshal/marshal.ts:21]: Will remove and have each Command Marshal inject it themselves if they need it

File: src/marshal/decorators/command.decorator.ts [hunk 3]
```diff
@@ -31,17 +26,29 @@ export const Command = (commandName: string | string[]): MethodDecorator => <T>(
...
+                        accountType: UserAuthenticationAccountType.DISCORD,
+                    },
+                    {
+                        id: true,
+                    },
+                ],
+            });
+            
+            const user = res.getUserByAuthAccount;
+            if (!user) throw new Error(`getUserByAuthAccount returned ${JSON.stringify(res)}`);
```
[the remark this answers is not in the document]
Comment (zachstence) [reply src/marshal/decorators/command.decorator.ts:43]: Possibly, I'll make it more generic

File: src/administrator-commands/sprocket-configuration.marshal.ts [hunk 4]
```diff
@@ -1,23 +1,46 @@
...
+                },
+            ],
+        });
+
+        const arr = configs.map(c => [c.key, c.value]);
+        const t = table(arr, ["key", "value"]);
+        await m.reply(`\`\`\`${t}\`\`\``);
+    }
+
+    // TODO do we support optional arguments? These could be the same command with key being optional
```
[the remark this answers is not in the document]
Comment (zachstence) [reply src/administrator-commands/sprocket-configuration.marshal.ts:28]: 👀

File: src/administrator-commands/sprocket-configuration.marshal.ts [hunk 5]
```diff
@@ -1,23 +1,46 @@
-import {CoreEndpoint, ResponseStatus} from "@sprocketbot/common";
 import {Message} from "discord.js";
+import {table} from "src/util/table";
```
[the remark this answers is not in the document]
Comment (zachstence) [reply src/administrator-commands/sprocket-configuration.marshal.ts]: 68f6e26

File: src/administrator-commands/sprocket-configuration.marshal.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (zachstence) [reply src/administrator-commands/sprocket-configuration.marshal.ts]: 94206df

File: src/marshal/decorators/command.decorator.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (zachstence) [reply src/marshal/decorators/command.decorator.ts]: 619417e

File: src/marshal/marshal.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (zachstence) [reply src/marshal/marshal.ts]: c7be6ca

**Review by 0c370t — APPROVED** [review]

**Pull request merged**